assets/js/common/ActionsButton/ActionsButton.stories.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { map, noop } from 'lodash';
+
+import ActionsButton from './ActionsButton';
+
+export default {
+  title: 'Components/ActionsButton',
+  component: ActionsButton,
+  argTypes: {
+    actions: {
+      description: 'Actions to be displayed in the actions button menu',
+      control: 'array',
+    },
+  },
+  args: {
+    actions: [
+      {
+        value: 'Operation 1',
+        running: false,
+        disabled: false,
+        onClick: noop,
+      },
+      {
+        value: 'Operation 2',
+        running: false,
+        disabled: false,
+        onClick: noop,
+      },
+    ],
+  },
+};
+
+export function Default(args) {
+  return <ActionsButton {...args} />;
+}
+
+export function Disabled({ actions }) {
+  return (
+    <ActionsButton
+      actions={map(actions, (action, index) =>
+        index === 0 ? { ...action, disabled: true } : action
+      )}
+    />
+  );
+}
+
+export function Running({ actions }) {
+  return (
+    <ActionsButton
+      actions={map(actions, (action, index) =>
```

Comment:
or even have a simple helper function ? 
```

const modifyAction = (actions, key, value) =>
  actions.map((action, index) =>
    index === 0 ? { ...action, [key]: value } : action
  );

export function Disabled({ actions }) {
  return <ActionsButton actions={modifyAction(actions, 'disabled', true)} />;
}

export function Running({ actions }) {
  return <ActionsButton actions={modifyAction(actions, 'running', true)} />;
}
```

Reply:
I will go with `Object.assign([], actions, {0: { ...actions[0], disabled: true }})`.
I don't want to create a function for a such a simple thing